src/main/groovy/edu/oregonstate/mist/locations/db/CachedFacilDAO.groovy

```diff
@@ -0,0 +1,73 @@
+package edu.oregonstate.mist.locations.db
+
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.databind.ObjectMapper
+import edu.oregonstate.mist.locations.Cache
+import edu.oregonstate.mist.locations.core.FacilLocation
+import groovy.transform.CompileStatic
+import org.skife.jdbi.v2.DBI
+import org.skife.jdbi.v2.exceptions.DBIException
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import java.sql.SQLException
+
+/**
+ * CachedFacilDAO is a wrapper around FacilDAO that adds caching.
+ */
+@CompileStatic
+class CachedFacilDAO implements Closeable {
+    final static Logger LOGGER = LoggerFactory.getLogger(CachedFacilDAO)
+    final ObjectMapper mapper = new ObjectMapper()
+
+    final private static String CACHE_FILENAME = "buildings.json"
```

Comment:
This could be misleading, since this doesn't contain all the building data (doesn't include coordinates or gender-inclusive restrooms). Maybe call it `facil-buildings.json`?

Reply:
Also, some of our other cached filenames are declared in the configuration file. I don't really want a mix of hardcoded filenames and configuration.yaml filenames, so I'd say lets choose one or the other. Which option do you prefer? I don't see a problem with hardcoding them, since I can't think of a reason the names would need to be different, let alone needing to be different based on the deployment environment.

Reply:
Agreed. I don't see any reason why they need to be configurable, since they all go in the cache directory and the cache directory is itself configurable. I'd like to move towards hardcoding them. I wanted to change the others to be hardcoded, but didn't quite get around to it for this PR (which is already fairly large). Perhaps as a separate ticket?

Reply:
Filed CO-1132